manifests/deploy/openshift/olm/bundle/4.6/meteringoperator.v4.6.0.clusterserviceversion.yaml

```diff
@@ -253,7 +253,7 @@ spec:
 
   keywords: [metering metrics reporting prometheus chargeback]
   version: "4.6.0"
-  minKubeVersion: "1.18.0"
+  minKubeVersion: "1.17.0"
```

Comment:
Not sure if this in unavoidable because of Helm, but it's bad to have to make changes like this in more than one place, it's error-prone and a dev burden.

Reply:
This is unavoidable because we're required to have a list of rendered YAML manifests that ART can consume when building our operator in a disconnected environment (OSBS). In general, though, that's what helm is doing, rendering YAML templating and spitting out rendered YAML manifests.

Reply:
✔️